views/index.ejs

```diff
@@ -119,6 +114,22 @@
       userName = $('#user-name').val().trim();
       userEmail = $('#user-email').val().trim();
 
+      if(currentUser != "/") {
```

Comment:
I'm a little bit confused here, how is that possible that the currentUser would be '/'?

Reply:
@cintiadr When I printed out currentUser onto the console, it turned out to be "/" when no user was logged in. Then again, I've no idea why it ended up like that in the first place. :/

Reply:
Super weird. I wouldn't bother for now, but I suppose at some point you might discover why is that and try to clean that up. 

Can you at least add a comment here? Maybe even a TODO comment.

Reply:
@cintiadr I just had a look at it. When no 'user' parameter is being passed to the page, value of hidden form element '#user-id' becomes "/" instead of "" as it should be (maybe it is some representation of null?). Maybe I should set the currentUser to "visitor" or something like that, is he isn't logged in?

Reply:
@cintiadr I changed the code to set currentUser as "visitor" when the user is logged, as there are a lot of places in the code where "visitor" is being used. I tested stuff to see if everything worked and so far, nothing broke.